src/libs/DistanceRequestUtils.js

```diff
@@ -67,13 +67,13 @@ function convertDistanceUnit(distanceInMeters, unit) {
  * @param {Number} rate Expensable amount allowed per unit
  * @param {String} currency The currency associated with the rate
  * @param {Function} translate Translate function
- * @returns {String} A string that describes the distance travled and the rate used for expense calculation
+ * @returns {String} A string that describes the distance traveled and the rate used for expense calculation
  */
 const getDistanceMerchant = (distanceInMeters, unit, rate, currency, translate) => {
     const convertedDistance = convertDistanceUnit(distanceInMeters, unit);
     const distanceUnit = unit === CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES ? translate('common.miles') : translate('common.kilometers');
     const singularDistanceUnit = unit === CONST.CUSTOM_UNITS.DISTANCE_UNIT_MILES ? translate('common.mile') : translate('common.kilometer');
-    const roundedDistance = convertedDistance.toFixed(2);
+    const roundedDistance = convertedDistance === 0 ? translate('common.tbd') : convertedDistance.toFixed(2);
```

Comment:
NAB: I don't really like that we're comparing against 0 here, I would rather add a `hasRoute` param and use that. What if we have identical start and stop waypoints, or is that validation handled earlier?